theme/src/resources/templates/article_pages/system_status.hbs

```diff
@@ -0,0 +1,153 @@
+<div class="container-fluid container-fluid-max-xl main-content">
+	<div class="row">
+		<div class="col-sm-12" id="systemStatusBanner"></div>
+
+		<div class="col-sm-12">
+			{{breadcrumbs}}
+		</div>
+	</div>
+
+	{{#with article}}
+	<div class="row">
```

Comment:
Each element should be indented 1 tab when it is nested within a handlebars template tag.  See https://github.com/rdai10/zendesk/blob/master/theme/src/resources/templates/article_page.hbs#L8...L80.

Reply:
8e352c9